### PR TITLE
Bugfix/pythonpath change

### PR DESCRIPTION
**Pull request by AndrewEichmann-NOAA**
Companion to https://github.com/NOAA-EMC/GDASApp/pull/518

Not much to say

### CODE REVIEW

File: test/soca/gw/CMakeLists.txt [hunk 1]
```diff
@@ -27,7 +27,7 @@ add_test(NAME test_gdasapp_soca_concatioda
          WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/test/soca/gw/concatioda)
        set_tests_properties(test_gdasapp_soca_concatioda
       PROPERTIES
-      ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/ush:$ENV{PYTHONPATH}")
+      ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/ush:${PROJECT_BINARY_DIR}/lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}:$ENV{PYTHONPATH}")
```
Comment (CoryMartin-NOAA) [inline test/soca/gw/CMakeLists.txt:30]: This will just fix the one test. What about the other SOCA tests that are failing?

Comment (AndrewEichmann-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: That looked like a different problem - is it not? I got this working last thing yesterday

Comment (CoryMartin-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: I'll be honest I don't know what the issue is for the other tests.

Comment (AndrewEichmann-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: nope, it's not

Comment (AndrewEichmann-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: Ok, now `test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP` works but `test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT` fails with 

 `0: slurmstepd: error: execve(): /scratch1/NCEPDEV/da/Andrew.Eichmann/fv3gfs/develop-develop/global-workflow/sorc/gdas.cd/../../sorc/gdas.cd/build/bin/soca_error_covariance_training.x: No such file or directory`
 
 @guillaumevernieres Were you mentioning this problem in a different context earlier?

Comment (CoryMartin-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: The executable has changed names I believe

Comment (guillaumevernieres) [reply test/soca/gw/CMakeLists.txt:30]: sorry @AndrewEichmann-NOAA , just getting around to looking into this. Yes, just a name change and maybe a yaml update. I'll push a fix in a bit.

Comment (AndrewEichmann-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: @guillaumevernieres @CoryMartin-NOAA It appears that the pythonpath update will have to be added to the prep jjob as well in order to work in cycling - this seems to be the issue @ShastriPaturi  is encountering first

Comment (CoryMartin-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: take a look at what I did in #518 , you can 'batch' update PYTHONPATH for multiple ctests with just 1 extra line per test in the CMakeLists.txt

Comment (AndrewEichmann-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: I tried that solution but it didn't seem to recognize the existing pythonpath.
With the addition from @guillaumevernieres all of the ctests pass, so I'd lobby to get this in.

Comment (AndrewEichmann-NOAA) [reply test/soca/gw/CMakeLists.txt:30]: @CoryMartin-NOAA Thanks for activating the tests

**Review by CoryMartin-NOAA — APPROVED** [review]
Thanks @AndrewEichmann-NOAA and @guillaumevernieres !

**Pull request merged**